build/better-go-playground.service

```diff
@@ -12,6 +12,7 @@ ExecStart=/opt/better-go-playground/playground -f /opt/better-go-playground/data
 Restart=on-failure
 LimitNOFILE=1048576
 RestartSec=5
+Environment="PATH=$PATH:/usr/local/go/bin"
```

Comment:
* Go should be already in `PATH` if it's installed, all distros create symlink to go binary in PATH.
* Go installation path differs in different distros.

Reply:
@avdb13 Debian is not the only distro. Users might use Ubuntu, Arch, Fedora or any other distro.

Anyway, by default Go binary is available at path and default systemd unit file provides a simple configuration that will work on most distros.

If user wants to install another version of Go from tarball directly - he still need to create a symlink to Go binary in `/usr/bin` or other directory in `PATH`.

Also sometimes there are cases when user might need to use multiple different versions of Go or use a wrapper like [gvm](https://github.com/moovweb/gvm).